src/components/JobParameterModal.vue

```diff
@@ -7,10 +7,8 @@
         </ion-button>
       </ion-buttons>
       <ion-title>{{ translate('Custom Parameters') }}</ion-title>
-      <ion-buttons slot="end" v-if="customRequiredParameters.length || customOptionalParameters.length">
-        <ion-button fill="clear" @click="copyToClipboard(getParameters(), 'Copied to clipboard')">
-          <ion-icon slot="icon-only" :icon="copyOutline" />
-        </ion-button>
+      <ion-buttons slot="end">
+        <ion-button color="primary" @click="save()">{{ translate('Save') }}</ion-button>
```

Comment:
Disable this button if the job is in pending state as per the discussion on the issue.